src/Microsoft.ML.Data/Transforms/ColumnConcatenatingTransformer.cs

```diff
@@ -212,22 +220,23 @@ internal ColumnInfo(ModelLoadContext ctx)
 
         private readonly ColumnInfo[] _columns;
 
-        public IReadOnlyCollection<ColumnInfo> Columns => _columns.AsReadOnly();
+        public IReadOnlyCollection<(string outputColumnName, string[] inputColumnNames)> Columns
```

Comment:
>public [](start = 8, length = 6)

should this be internal ?

Reply:
I think the general trend is to keep a `Columns` property in the transformers so that people can have access to the input output columns that they are acting on.

---
In reply to: [253985146](https://github.com/dotnet/machinelearning/pull/2363#discussion_r253985146) [](ancestors = 253985146)